pkg/fileservice/bytes.go

```diff
@@ -80,8 +85,7 @@ func (b *bytesAllocator) allocateCacheData(size int, hints malloc.Hints) fscache
 		bytes:       slice,
 		deallocator: dec,
 	}
-	bytes._refs.Store(1)
-	bytes.refs = &bytes._refs
+	bytes.refs.Store(1)
```

Comment:
Minor, you can do a NewBytes here, so that we can restrict all the Store(1) to one single place.